provisioning/ansible/roles/beetbox-init/tasks/main.yml

```diff
@@ -55,3 +55,30 @@
     line: "SSH_HOME={{ beet_ssh_home }} && [ -e $SSH_HOME ] && cd $SSH_HOME"
   become: no
   when: beet_ssh_home is defined
+
+- name: Set PATH for Composer vendor binaries.
+  lineinfile:
+    dest: "/home/{{ beet_user }}/.bashrc"
+    state: present
+    create: yes
+    regexp: "^\\s*export PATH=\"{{ path_item | regex_escape() }}:\\$PATH\""
+    line: |
+      # Add Composer vendor binaries to PATH.
+      if [ -d "{{ path_item }}" ]; then
+        export PATH="{{ path_item }}:$PATH";
+      fi
+    owner: "{{ beet_user }}"
+    group: "{{ beet_user }}"
+    mode: 0644
+  with_first_found:
+    - files:
+      - "vendor/bin"
+      paths:
+      - "{{ beet_base }}"
+      - "{{ beet_root }}"
+      - "{{ beet_web }}"
+      - "{{ composer_home_path }}"
```

Comment:
@christopher-hopper thanks for the PR -- I have an existing PR open on the composer role which would address this -- https://github.com/geerlingguy/ansible-role-composer/pull/26

It would be my preference to include it in the upstream role (+1 if you want) but if we can't get it included, we may need to fork this role as any composer related tasks should be in the composer role.

To provide a bit of background, I've been trying to remove as many tasks/roles from the core project and move them into external roles -- http://beetbox.readthedocs.io/en/latest/contributing/contributing/

The goal is to minimise the support requirements on the core project and push features back to roles which can be maintained externally.